src/main/java/com/aws/greengrass/deployment/DeploymentDocumentDownloader.java

```diff
@@ -223,31 +235,32 @@ private GetDeploymentConfigurationResponse getDeploymentConfiguration(String dep
         } catch (SdkClientException e) {
             throw new RetryableDeploymentDocumentDownloadException(
                     "Failed to contact Greengrass cloud or unable to parse response", e);
-        }  catch (TLSAuthException e) {
-            throw new RetryableClientErrorException(
-                    "Failed to contact Greengrass cloud or unable to parse response", e);
+        } catch (TLSAuthException e) {
+            throw new RetryableClientErrorException("Failed to contact Greengrass cloud or unable to parse response",
+                    e);
         }
         return deploymentConfiguration;
     }
 
     private void validateHttpExecuteResponse(HttpExecuteResponse executeResponse)
             throws RetryableDeploymentDocumentDownloadException, DeploymentTaskFailureException {
         if (!executeResponse.httpResponse().isSuccessful()) {
-            throw new RetryableDeploymentDocumentDownloadException(String.format(
-                    "Received unsuccessful HTTP status: [%s] when getting from preSigned url. Status Text: '%s'",
-                    executeResponse.httpResponse().statusCode(),
-                    executeResponse.httpResponse().statusText().orElse(StringUtils.EMPTY)));
+            throw new RetryableDeploymentDocumentDownloadException(String
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses &#39;%s&#39; to format int: &#39;statusCode&#39; expression. This is a potential locale-sensitive handling issue. It might cause errors in the handling and processing of the statement at line: 248. Consider formatting this data with &#39;%d&#39; instead.